packages/flutter/lib/src/services/autofill.dart

```diff
@@ -799,13 +799,36 @@ mixin AutofillScopeMixin implements AutofillScope {
       !autofillClients.any((AutofillClient client) => client.textInputConfiguration.autofillConfiguration == null),
       'Every client in AutofillScope.autofillClients must enable autofill',
     );
-    return TextInput.attach(
-      trigger,
-      _AutofillScopeTextInputConfiguration(
-        allConfigurations: autofillClients
-          .map((AutofillClient client) => client.textInputConfiguration),
-        currentClientConfiguration: configuration,
-      ),
+
+    TextInputConfiguration inputConfiguration;
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.iOS:
+        bool isPassword(AutofillClient client) {
+          final Iterable<String> hints = client.textInputConfiguration.autofillConfiguration?.autofillHints;
+          return client.textInputConfiguration.obscureText
+            || (hints?.contains(AutofillHints.password) ?? false)
+            || (hints?.contains(AutofillHints.newPassword) ?? false);
+        }
+
+        // In a native iOS app, Only password fields that have multi-field
```

Comment:
Lowercase "o" on "Only" here.

Also I think you forgot a word after that. "save"?